src/components/reusable-components/searcher/form/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styles from "./styles.module.scss"
+
+class Form extends React.Component{
+
+    render(){
+        return(
+            <form action="" className={styles.buscador}>
+                <div className={styles.lupa1}>
+                    <img src="./back.png" className={styles.lupa} alt=""/>

```

Comment:
Te recomiendo abstraer un poco el nombre de las clases para que no se hable propiamente de lo que se ve si no de lo que representa, así será más fácil para quien use el código después, entender de qué se habla:
```suggestion
                <div className={styles.contenedorIcono}>
                    <img src="./back.png" className={styles.icono} alt=""/>
```